src/components/Users/CreateUserForm.tsx

```diff
@@ -127,7 +122,7 @@ export default function CreateUserForm({ onSubmitSuccess }: Props) {
     }
   }, [phoneNumber, isWhatsApp, form, usernameInput]);
 
-  const { error, isLoading } = useQuery({
+  const { isLoading, isError } = useQuery({
```

Comment:
since this is not the primary API of this component, either alias the isLoading, isError, or skip deconstructing it.
```suggestion
  const usernameCheckQuery = useQuery({
```

Reply:
Let's go with alias